locations/spiders/gamestop_ca.py

```diff
@@ -0,0 +1,25 @@
+import scrapy
+
+from locations.items import Feature
+
+
+class GamestopSpider(scrapy.Spider):
+    name = "gamestop_ca"
+    item_attributes = {"brand": "GameStop", "brand_wikidata": "Q202210"}
+    allowed_domains = ["www.gamestop.com"]
+    start_urls = ["https://www.gamestop.ca/StoreLocator/GetStoresForStoreLocatorByProduct?value=&language=en-CA"]
```

Comment:
Looks like all the stores except the US are standard.

https://www.gamestop.ca/StoreLocator/GetStoresForStoreLocatorByProduct
https://www.gamestop.de/StoreLocator/GetStoresForStoreLocatorByProduct
https://www.gamestop.ie/StoreLocator/GetStoresForStoreLocatorByProduct
https://www.gamestop.it/StoreLocator/GetStoresForStoreLocatorByProduct
https://www.gamestop.ch/StoreLocator/GetStoresForStoreLocatorByProduct

Reply:
Also it looks like they do something weird with `"undefined"`, something like the following should normalise it.

```python
for key, value in data.items():
    if value == "undefined":
        data[key] = None
```

Reply:
Well you noticed it in the lat/lon fields, it's also in some of the address fields. If you don't mind, I'll push to this PR, just make sure you do a pull, a force push would throw away my commit.